apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -469,7 +469,9 @@
       "dydx_title": "Ledger Nano S™ does not support swapping dYdX",
       "dydx_description": "To swap dYdX, use any other compatible Ledger device, such as the Ledger Nano S Plus™, Ledger Nano X™, Ledger Flex™ or Ledger Stax™.",
       "sui_title": "Ledger Nano S™ does not support swapping Sui",
-      "sui_description": "To swap Sui, use any other compatible Ledger device, such as the Ledger Nano S Plus™, Ledger Nano X™, Ledger Flex™ or Ledger Stax™."
+      "sui_description": "To swap Sui, use any other compatible Ledger device, such as the Ledger Nano S Plus™, Ledger Nano X™, Ledger Flex™ or Ledger Stax™.",
+      "kaspa_title": "Ledger Nano S™ does not support swapping Kaspa",
+      "kaspa_description": "To swap Kaspa, use any other compatible Ledger device, such as the Ledger Nano S Plus™, Ledger Nano X™, Ledger Flex™ or Ledger Stax™."
```

Comment:
Just leave this file. All the translations to the other languages are done automatically.